tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -465,6 +465,13 @@ static void thread_mbox_data_get_null(void *p1, void *p2, void *p3)
  */
 void test_mbox_data_get_null(void)
 {
+#if defined(CONFIG_CPU_ARCEM) || defined(CONFIG_CPU_CORTEX_M)
```

Comment:
The comment explains why ARCEM, but why CPU_CORTEX_M here?

Reply:
For Cortex-M, it can read access to address 0x0 is generally allowed. The details in commit: f620ceacde